lib/spack/spack/cmd/external.py

```diff
@@ -146,7 +147,16 @@ def external_find(args):
         packages_to_check = spack.repo.path.all_packages()
 
     pkg_to_entries = _get_external_packages(packages_to_check)
-    _update_pkg_config(pkg_to_entries, args.not_buildable)
+    new_entries, write_scope = _update_pkg_config(
+        pkg_to_entries, args.not_buildable
+    )
+    if new_entries:
+        msg = ('The following specs have been detected on this system '
+               'and added to "packages.yaml" [scope={0}]')
```

Comment:
I would just put the real path to the file here.

Reply:
That's what we do for `spack compiler find` too, best to be consistent.

Reply:
Done in 19d629c